mod/wikirate/set/all/wikirate.rb

```diff
@@ -226,7 +226,7 @@ def enrich_result result
 
   def main_name
     left_name = card.cardname.left_name
-    left_name = left_name.left unless card.name.include?("limited metric")
+    left_name = left_name.left unless card.key.include?("limited_metric")
```

Comment:
We should never see a card's name or key in code.  "limited metric" needs a codename.